recipes/sdl/all/CMakeLists.txt

```diff
@@ -2,14 +2,11 @@ cmake_minimum_required(VERSION 3.0)
 project(cmake_wrapper)
 
 include(${CONAN_INSTALL_FOLDER}/conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
+conan_basic_setup(TARGETS)
```

Comment:
I"m not sure this is correct for older SDL releases.

Reply:
I revert this type of things to current in CCI